### DISCUSSION TITLE
more disinfectants

### DISCUSSION THREAD

**Discussion by robotic-gamer** [CleverRaven/Cataclysm-DDA#48418] [Ideas]
-a lot more substances have been used irl to disinfect wounds that could be used.
-given the improvised nature they could be given a low (60%) chance to treat bites
-it would still make them a better option then cauterizing
-the ones I could think of:
>strong alcohol like vodka
>vinegar
>phenol (could be made by distilling willow bark)
>turpentine (could be byproduct of making rosin)
>wood vinegar (impure vinegar made by distilling wood)
>bleach
>in theory any reasonably strong acid could be watered down and used
>coal tar (carcinogenic but has painkilling properties)
>tannic acid (from tanbark)

edit: in the short term
add a "crude disinfectant" type with a 60-80% chance to treat bites
substances that could be used directly include:
- vinegar
- vodka, whiskey other "strong drinking alcohol"

substances that need watering down first:
- acetic acid
- acetone
- bleach
- oxidizer powder
- phenol

**Comment by Entity-y** [comment]
Antiseptic mouthwash might also work in a pinch.

**Comment by actual-nh** [comment]
A couple of oddities I noticed while exploring around with the item browser:
- Antiseptic/disinfectant has both a recipe and IRREPLACEABLE_CONSUMABLE.
- Potassium iodide tablets can be crafted from antiseptic (or iodized salt; OK) + aspirin; antiseptic can be crafted using oxidizer powder + water + alcohol; oxidizer powder which can be crafted from saline solution (or salt water), ammonia, and hydrochloric acid. Umm... where's the iodine coming from? I'm not aware of medical saline solution having iodine.

Regarding the strong alcohol mentioned - see "makeshift antiseptic".

**Reply by robotic-gamer** [reply]
- by strong alcohol I was referring to the kind of alcohol that can be used to make "strong mixed alcohol" like vodka, whiskey and rum the (40ish % stuff). not the 95% stuff in ethanol.
-the idea is the less pure alcohol is still a decent disinfectant that would be a step up from cauterizing.
-it could be made bulkier and only have 60% chance to cure bites to reflect its lower grade nature

-I am guessing they are interpreting antiseptic to be iodine based when crafting potassium iodide
-but interpreting antiseptic to be a mix of alcohol and oxidizer powder when crafting antiseptic

edit:formating

**Reply by actual-nh** [reply]
70% is actually sufficient to kill bacteria reasonably killable with ethanol (ones that are in spore form or osmotolerant, as in capable of living in high salt solutions or high sugar solutions - including honey - or in a desert, can be resistant), and 62-63% in a gel form (less water in it). This will also kill/destroy enveloped viruses (ones with a membrane around them) - important given that coronaviruses are enveloped - and most (not all, just as with bacteria!) fungi. Non-enveloped viruses can be harder.

BTW, radiation-tolerant bacteria (_Deinococcus radiodurans_, for instance) usually have gotten that way from resisting dehydration (it does similar DNA damage; that particular one is from a desert). So something in a radioactive crater may well be resistant to alcohol, high salt, and high sugar. (And yes, something could be there - in a graduate school seminar on extremophiles, I presented (someone else's) research paper on a biofilm they found growing inside the containment vessel of a nuclear power plant, on high-grade stainless steel "fins" in the purified water around the core. It was rather of a problem to study because it was so radioactive they couldn't take it out of the containment vessel...)

Less than 70% purity alcohol... combine it with something else if you can. Say, as much salt as you can dissolve into it - although that's going to give you a massive pain penalty. (Any of the antiseptics on a wound also tend to promote scarring, BTW.) Or as much sugar as you can get to dissolve in it, although it's likely to be less effective (less can dissolve in it). The key is a low "water activity".

I'll have to take a look at any current recipes to concentrate alcohol - freezing will work, actually, or a vacuum oven, not just a still (they are more efficient, however).

Regarding the iodine - two forms of antiseptic, probably using the `id_suffix` capability, seem to be needed.

EDIT: Oh - @Venera3 - any further comments from a surgical/etc perspective? Thanks!

**Reply by robotic-gamer** [reply]
you seem to be better at understanding at the underlining biology then me so correct me if I am wrong but
-its not necessary to sterilize the wound with the disinfectant merely kill most of the potential pathogens to prevent an infection
-its not necessary to kill viruses or fungi(evident by the usage of sulfa drugs which only work on bacteria in ww2 to treat wounds)
-I doubt the radiation tolerant bacteria are the same ones to be pathogenic
-my basis for thinking vinegar and vodka could work as disinfectants is that they are strong enough to preserve food

-my understanding of freeze distillation is it can only get you to the strength of vodka (40%). this is based on the wiki entry for "Applejack" a type of freeze distilled apple cider
-there should probably also be a difference between pre-cataclysm iodized salt and post cataclysm non-iodized salt. the iodine in the former could easily be extracted via distillation

edit: the is also the large number of scenarios in media that show people using drinking whiskey/vodka for wounds

**Comment by Venera3** [comment]
The thing about cleaning something like a bite is *what* you clean it with is is much less important than *how* you clean it. As long as your cleaning agent is relatively bacteria-poor, water-based and tissue friendly (that part is where alcoholic hand sanitizers and the like falls down hard, by the way - don't clean a wound with them IRL), you have enough to get all the gunk out and can get at all the little nooks and crannies you're good.

In that sense yes, there should be a lot more potential wound cleaning agents, but wound care should also be *way* more difficult and require either high health care levels and/or its own proficiency. Doing it justice also requires a way to track sterility of the used materials, and a way for non-food to spoil into an arbitrary item (ie boiled bandages spoiling into normal ones after an hour or so), off the top of my head.

**Reply by actual-nh** [reply]
Point. It's sort of like handwashing - by no means is its effectiveness entirely determined by what you're using, it's whether you get between the fingers and whether you do enough physical rubbing/friction to take off stuff. (This is from running handwashing experiments with students 2-3 times a year for quite a few years - swab hand for stuff, put on one side of petri dish, clean hand, swab hand again, put on other side of petri dish, incubate. The most effective of all I've found is unfortunately impractical - mechanic's soap (with pumice in it), wash it completely off with water, followed by alcohol gel... which would rub your hands raw in no time.)

Does the above differ if you're talking about an already-obviously-infected wound vs one that may be infected? (And I anticipate it could differ yet again for an obviously-infected wound if there was much necrotic - already-dead - tissue in the area.)

**Reply by actual-nh** [reply]
Liquid alcohol, BTW, does have a definite role to play in sterilizing metal instruments, if they can take it - dip them in it then flame them off.

**Reply by Coseo** [reply]
> In that sense yes, there should be a lot more potential wound cleaning agents, but wound care should also be way more difficult and require either high health care levels and/or its own proficiency.

Wound debridement and disinfection is a lot more annoying and difficult to have to do on yourself than have someone else do for you, but isn't something that's so incredibly difficult that having a "high level of health care skill" is required to do it at all, let alone with a decent chance of success. If you're _really_ arguing for a hyper realistic approach, then just about any wound can get infected regardless of depth and difficulty of cleaning should be decided by the nastyness of the wound - at the lowest level, just about any human being that has ever had a cut before in the modern world has likely gotten some experience cleaning it out with disinfectant. Even at the upper level, this isn't the sort of thing you need a 4 year degree to be able to pull off in real life - as someone who lives with an individual who just finished schooling in a medical field, watched them literally learning about the topic and practicing and getting their certifications, and also sees them regularly doing this for an infected wound our cat has, it's not only not that difficult, but there's a lot of techniques you can do to make it easier - especially for shallower wounds or ones that don't have complex inner areas you have to clean out. 

In fact, I actually asked the person who does this (My sister, a Vet Tech) about this, and she told me that disinfecting wounds and initial wound care is trivial enough that she'd trust most people to get it right even untrained - after that, basic wound care that covers most scenarios is simple enough that she was taught it over the course of a _single day_, and she'd be confident giving basic instructions on how to debride wounds, even deeper puncture ones, to a layman without training and expect them to be able to do it in most scenarios.

Furthermore, most medical manuals cover this sort of thing, and in an apocalypse filling a syringe with betadine or a homemade cleaning solution can be enough to flush out even deeper puncture wounds. 

From a gameplay perspective, having literally any infected wound be a death sentence unless you have "high medical skill" just sounds awful to me. 

At least, that's my two cents.

**Comment by ActualDio** [comment]
I really think you should be able to use soapy water as a disinfectant, provided you also have clean water as well and it takes longer to do. Soap is incredibly good at making things soluble in water and in most cases, is better than alcohol if Im not mistaken, provided you use clean water to wash off the wound after. 
This might of course make it much easier to deal with wounds and cripple the difficulty there but its way more realistic to the actual power of soap.
It shouldnt, I dont think, be able to treat an infection, if the wound is already infected because then the soap cant really do much besides removing the surface level germs

**Reply by coyo7e** [reply]
Yeah, using soapy water on wounds is more of a way to clean it of debris, not to stop an existing infection once it's begun. I got an infection from road rash once and had to use a prescription strength sulfur-based topical "burn cream" on it once it turned green and started to stink like rotten meat (yeah sorry it's gross, but infections are pretty gross), the cream was designed to allow the wound to continue to "Weep" and push out contagions and stuff while protecting it from open air contact, and killing off the actual bacteria causing the infection.

**Comment by coyo7e** [comment]
Bleach is dangerous as hell - it's extremely caustic and even making skin contact with it for a split second will leave you with burnt skin that'll deaden and fall off. Pouring it onto a wound would be horrific.
